src/background/actions/assessment-action-creator.ts

```diff
@@ -229,6 +234,14 @@ export class AssessmentActionCreator {
         this.telemetryEventHandler.publishTelemetry(TelemetryEvents.SELECT_REQUIREMENT, payload);
     };
 
+    private onSelectGettingStarted = (payload: SelectGettingStartedPayload): void => {
+        this.assessmentActions.selectGettingStarted.invoke(payload);
```

Comment:
You can actually use the subview action here with the populated property of getting started. I would probably use the selected subview payload in the details view action message creator so you only have to change the action here.